tests/test_flows.py

```diff
@@ -317,7 +317,7 @@ def test_dev_reset_default(self):
         self.check_no_cmdu_type("autoconfig search while in reset", 0x0007, agent.mac)
         env.checkpoint()
         agent.cmd_reply("dev_set_config,backhaul,eth")
-        time.sleep(1)
+        time.sleep(3)
```

Comment:
I did similar change on my PR.
https://github.com/prplfoundation/prplMesh/pull/1593/commits/578a2141ca98832e595d1e2570f904e519272f35

Reply:
Apparently your hardware is better than mine. I need two extra seconds and you need only one :)